modules/draco/src/lib/draco-module-loader.js

```diff
@@ -21,7 +21,11 @@ export async function loadDracoDecoderModule(options) {
 
   // Check if a bundled draco3d library has been supplied by application
   if (modules.draco3d) {
-    loadDecoderPromise = loadDecoderPromise || modules.draco3d.createDecoderModule({});
+    loadDecoderPromise =
+      loadDecoderPromise ||
+      modules.draco3d.createDecoderModule({}).then(draco => {
+        return {draco};
```

Comment:
Nit: I remember there being a comment about why we needed to wrap this return value inside an object. The returned module looked like a Promise so we needed to wrap it to avoid other code awaiting on that value. This was a big gotcha and the helpful comment seems to have been lost.